app-shells/fish/fish-2.4.0.recipe

```diff
@@ -59,8 +59,14 @@ GLOBAL_WRITABLE_FILES="
 BUILD()
 {
 	autoreconf -vfi
-	CFLAGS=-D_BSD_SOURCE LDFLAGS="-lnetwork" runConfigure ./configure \
+
+	export CFLAGS="-D_BSD_SOURCE"
+	export CXXFLAGS=$CFLAGS
+	export LIBS="-lnetwork -lexecinfo"
```

Comment:
You should amend configure.in with something like:
AC_SEARCH_LIBS(socket, socket network)
AC_SEARCH_LIBS(backtrace, execinfo)
and remove LDFLAGS from the recipe.